src/components/SignIn/useSignIn.ts

```diff
@@ -0,0 +1,38 @@
+import axios from "../../utils/axios";
+import { useState } from "react";
+import { useRouter } from "next/router";
+import { useToast } from "@chakra-ui/react";
+
+export default function useSignIn() {
+  const [isLoading, setIsLoading] = useState<boolean>(false);
```

Comment:
`<boolean>` is unnecessary since it can be inferred from your initial value `false`.

Reply:
done